debug_toolbar/panels/__init__.py

```diff
@@ -20,24 +20,23 @@ def panel_id(self):
         return self.__class__.__name__
 
     @property
-    def enabled(self):
+    def enabled(self) -> bool:
+        # The user's cookies should override the default value
+        cookie_value = self.toolbar.request.COOKIES.get("djdt" + self.panel_id)
```

Comment:
If a user has enabled a panel, then changes their settings file to disable it, wouldn't this override the customized settings?

Reply:
If I understand correctly, that's the same behavior  we have today, as the `default` value is only used when a cookie value is not found in the request.

This shouldn't change how a panel status is decided, just to return faster when the cookie is set.

Reply:
You're correct. `DebugToolbar.get_panel_classes` is what I was worried about. I wanted to make sure that a developer was still able to stop using a panel even if the cookies are set.